src/components/members/user.tsx

```
@@ -1,29 +1,252 @@
+            
+            {selectedMember.team.length > 0 ? (
+              <Stack spacing="xs">
+                {selectedMember.team.map((team) => (
+                  <Group 
+                    key={team} 
+                    position="apart" 
+                    p="xs" 
+                    style={{ 
+                      backgroundColor: colorScheme === 'dark' ? 'rgba(44, 46, 51, 0.5)' : 'rgba(241, 243, 245, 0.7)',
+                      borderRadius: '6px'
+                    }}
```

Biome (1.9.4)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)